pyflink-walkthrough/docker-compose.yml

```diff
@@ -47,6 +47,10 @@ services:
       - JOB_MANAGER_RPC_ADDRESS=jobmanager
   zookeeper:
     image: wurstmeister/zookeeper:3.4.6
+    ulimits:
+      nofile:
+        soft: 65536
+        hard: 65536
```

Comment:
Without that, that (old) Zookeeper version won't start on current Docker installations, which give unlimited file handles, causing the app to run out of memory. Ideally, we'd use a more up-to-date Zookeeper version, but there's no newer version of that Wurstmeister image, and I don't want to further expand scope of this change.